lib/uptime/env.ex

```diff
@@ -92,4 +146,32 @@ defmodule Uptime.Env do
   defp valid_env_var_value(value) do
     value != nil and value != ""
   end
+
+  # Values may be stringified (from env) or the correct type (from config file)
+  defp parse_value(value, :string), do: value
+  defp parse_value(nil, :integer), do: nil
+  defp parse_value(value, :integer) when is_integer(value), do: value
+  defp parse_value(value, :integer), do: String.to_integer(value)
+
+  defp parse_value(value, :boolean) when is_boolean(value), do: value
+  defp parse_value(value, :boolean) when value in [1, true], do: true
```

Comment:
I think you actually want "true" because true is a boolean and would get caught by the previous check

Reply:
oops you're right